README.md

```diff
@@ -245,6 +245,39 @@ Travis already have a provider for deploying to bintray, but it is extremely
 inconvenient as it requires to produce one json file per file to upload.
 
 
+Codecov
+-------
+
+Beaver includes a convenient wrapper to send [codecov.io](https://codecov.io/)
+reports. Just make sure the project was already built with coverage support and
+then use something like this in the `.travis.yml` file:
+
+```yml
+after_success: BEAVER_CODECOV_REPORTS="*.lst" beaver codecov [OPTIONS]
+```
+
+The `[OPTIONS]` are forwarded directly to the
+[codecov-bash](https://github.com/codecov/codecov-bash), but this script will
+run in a confined docker instance where only the coverage reports are available,
```

Comment:
So I wouldn't be able to just upload cached reports from a previous `travis` build stage?

Reply:
Why not?

Reply:
>  but this script will run in a confined docker instance where only the coverage reports are available,

As I understand from the description a new docker container will be started and the codecov bash script will be run inside that. OAIMS?

Reply:
That is correct. And why do you think you would not be able to use cached reports?